src/tb_loader.cpp

```diff
@@ -115,6 +121,22 @@ bool TBLoader::get_filter_nearest()
 	return m_filter_nearest;
 }
 
+void TBLoader::set_clip_texture_name(String clip_texture_name) {
+	m_clip_texture_name = clip_texture_name;
+}
+
+String TBLoader::get_clip_texture_name() {
+	return m_clip_texture_name;
+}
+
+void TBLoader::set_skip_texture_name(String skip_texture_name) {
+	m_skip_texture_name = skip_texture_name;
+}
+
+String TBLoader::get_skip_texture_name() {
+	return m_skip_texture_name;
+}
+
```

Comment:
`{` should be on a new line here. Also I think we can pass the strings as `const String&`, no?